bool isPaid
int32 carType
---
bool isAllowed